src/Foliage-Server/FOServer.class.st

```diff
@@ -0,0 +1,51 @@
+Class {
+	#name : #FOServer,
+	#superclass : #Object,
+	#instVars : [
+		'server'
+	],
+	#category : #'Foliage-Server'
+}
+
+{ #category : #accessing }
+FOServer class >> start [
+	<script>
+
+	^ self new start
+]
+
+{ #category : #accessing }
+FOServer class >> stop [
+	<script>
+	
+	^ ZnServer default stop
+]
+
+{ #category : #accessing }
+FOServer >> baseDirectory [
+
+	^ 'html' asFileReference
```

Comment:
we is that "html" when the target path above is "generated"?

Reply:
oops